packages/wp-now/src/tests/mode-examples/wordpress-with-config/wp-login.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+// File needs to exist to avoid throwing a 404 error
+
+header('Location: /wp-admin/');
```

Comment:
This new `wp-login.php` file is necessary to avoid this new exception on the blueprint step:
https://github.com/WordPress/wordpress-playground/blob/a8690c947996684a4d1df18061beb1ff1a3f927d/packages/playground/blueprints/src/lib/steps/login.ts#L53-L62